src/controllers/diaries.js

```diff
@@ -0,0 +1,161 @@
+var Diary = require('../models/Diary');
+
+var User = require('../models/User');
+
+exports.createDiaryEntry = function(req, res) {
+    User.findByToken({
```

Comment:
Since `findByToken` is used repeatedly, maybe we extract it into a middleware function and pass it to the router to authenticate before executing the diary controller method?

Would be easier to maintain - maybe if we have time, I'll try to refactor then.